src/pages/docs/troubleshooting/salesforce-testing/most-common-issues.md

```
@@ -131,4 +137,65 @@ Users can sync metadata but encounter an MFA challenge during the process.
 
+---
+
+## **Execution fails on Labs**
+
+### **Cause:**
+
+The execution fails when the session is generated from a different machine but the execution is done on a different browser or environment, such as when a lab is selected. This mismatch results in Salesforce treating the session as invalid, leading to execution failure.
+
+### **Steps to Resolve**
+
+**1. Verify User Privileges**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)